R/utils_checkmate.R

```diff
@@ -125,7 +127,11 @@ assert_df_with_variables <- checkmate::makeAssertionFunction(check_df_with_varia
 check_valid_factor <- function(x,
                                min.levels = 1,
                                max.levels = NULL,
-                               any.missing = TRUE) {
+                               empty.levels.ok = TRUE,
+                               null.ok = TRUE,
+                               any.missing = TRUE,
+                               n.levels = NULL,
```

Comment:
```suggestion
                               empty_levels_ok = TRUE,
                               null_ok = TRUE,
                               any_missing = TRUE,
                               n_levels = NULL,
```
Fix linter errors by using snake_case instead of `.` in variable names

Reply:
This was not causing the linter errors. I think it is the missing \cr in the docs.  We need these to have the dots to inherit parameters

Reply:
No, you were right. Still doing that! I added `#nolint` so we can inherit parameters w/o rewriting them.